sktime/param_est/base.py

```diff
@@ -155,20 +171,17 @@ def fit(self, X):
         Writes to self:
             Sets self._is_fitted flag to True.
             Writes `X` to self._X.
+            Writes `y` to self._y if y is not None.
             Sets fitted model attributes ending in "_".
 
         Parameters
         ----------
-        X : time series in sktime compatible data container format
-                Time series to which to fit the forecaster in the update.
-            y can be in one of the following formats, must be same scitype as in fit:
-            Series scitype: pd.Series, pd.DataFrame, or np.ndarray (1D or 2D)
-            Panel scitype: pd.DataFrame with 2-level row MultiIndex,
-                3D np.ndarray, list of Series pd.DataFrame, or nested pd.DataFrame
-            Hierarchical scitype: pd.DataFrame with 3 or more level row MultiIndex
-            For further details:
-                on usage, see forecasting tutorial examples/01_forecasting.ipynb
-                on specification of formats, examples/AA_datatypes_and_datasets.ipynb
+        X : sktime-compatible container
```

Comment:
can you leave the references on the permissible formats?

This may have to be updated though with newer API references, see the docstring for `BaseForecaster.fit`